pkg/server/health/health.go

```diff
@@ -29,6 +30,9 @@ func (o *Checker) AuthFuncOverride(ctx context.Context, fullMethodName string) (
 }
 
 func (o *Checker) Check(ctx context.Context, req *healthv1pb.HealthCheckRequest) (*healthv1pb.HealthCheckResponse, error) {
+	if o.SilentHealthChecks {
```

Comment:
Can you verify whether the intend is to suppress the log or suppress the healthcheck itself?

Reply:
Sorry, I don't understand the change in this file. We don't want to suppress the healthcheck itself, just the logs, right?